aiida_orca/parsers/__init__.py

```diff
@@ -75,11 +74,17 @@ def _remove_nan(parsed_dictionary: dict) -> dict:
         output_dict = _remove_nan(parsed_dict)
 
         if parsed_dict.get('optdone', False):
-            with out_folder.open(fname_relaxed, 'rb') as handle:
-                with tempfile.NamedTemporaryFile('w+b', suffix=pathlib.Path(fname_relaxed).suffix) as tmpfile:
-                    shutil.copyfileobj(handle, tmpfile)
-                    tmpfile.flush()
-                    relaxed_structure = StructureData(pymatgen_molecule=Molecule.from_file(tmpfile.name))
+            # Change this when we drop AiiDA 1.x support
+            #with out_folder.base.repository.open(fname_relaxed) as handler:
+            with out_folder.open(fname_relaxed) as handler:
+                ase_structure = ase.io.read(handler, format='xyz', index=0)
```

Comment:
Nitpick alert: usually it is referred to as a file-handle
```suggestion
            with out_folder.open(fname_relaxed) as handle:
                ase_structure = ase.io.read(handle, format='xyz', index=0)
```

Reply:
Good point, I was confused by this before. I've applied your suggestion throughout.